tests/trainer/test_fsdp_checkpoint.py

```diff
@@ -311,6 +318,9 @@ def test_fsdp_mixed_with_sync(world_size, tmp_path: pathlib.Path, sync_module_st
     pytest.param('0.14.1',
                  marks=pytest.mark.filterwarnings(
                      r'ignore:MosaicMLLogger is not in the state_dict. Its state will not be restored.:UserWarning')),
+    pytest.param('0.15.1',
```

Comment:
we should add 0.16 here, yeah?

Reply:
I will just add a 0.16.1 after this. It's kinda broken right now anyways...?

Reply:
but you added code to support checkpoints saved with 0.16.0